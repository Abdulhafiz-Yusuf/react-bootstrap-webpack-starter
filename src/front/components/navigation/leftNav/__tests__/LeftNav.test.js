// @flow

// #region imports
import React from 'react';
import renderer from 'react-test-renderer'; // needed both for snpashot testing but also to prevent errors from enzyme
import { MemoryRouter } from 'react-router';
import LeftNav from '../LeftNav';
// #endregion

describe('LeftNav component', () => {
  it('renders as expected', () => {
    const props = {
      link: '/',
      label: 'home',
      viewName: 'home',
      onClick: () => {},
    };

    const component = renderer
      .create(
        <div>
          <MemoryRouter>
            <LeftNav {...props} />
          </MemoryRouter>
        </div>,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
