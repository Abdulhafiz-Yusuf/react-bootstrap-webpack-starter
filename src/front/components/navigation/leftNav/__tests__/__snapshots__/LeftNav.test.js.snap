// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftNav component renders as expected 1`] = `
<div>
  <ul
    className="nav navbar-nav"
  />
</div>
`;
